Documentation/Signoffs/Power_System_Camera.md

```diff
@@ -1,37 +1,125 @@
 # Power System for Camera System
 ## Functionality
+The primary functionality for the camera power system is to reliably meet the power requirements for the camera hardware system's components. The system will supply 5 V to the camera system's Raspberry Pi 5 via a USB-C connection and 5 V to the Raspberry Pi HQ camera. The system will supply 5 V to both servo motors +5V DC and GND terminals.
 
 ## Constraints
 | No.| Contraint | Origin |
 | -- | --------- |--------|
-|  1 |           |        |              
-|  2 |           |        |                          
-|  3 |           |        |              
+|  1 | The system shall comply with ASTM B258-18 [2]. | Standards and Safety |
```

Comment:
ASTM B258-18 does not address current capability. Also footnote number should be 1, not 2.

Reply:
The new standard was added, and the references section was added for the in text citation.